integrations/livepeer/livepeer-client.tsx

```diff
@@ -0,0 +1,13 @@
+import { createReactClient, studioProvider } from '@livepeer/react'
+
+import { env } from '@/env.mjs'
+
+if (!env.NEXT_PUBLIC_LIVEPEER_API_KEY) {
+  throw new Error('No NEXT_PUBLIC_LIVEPEER_API_KEY provided')
+}
+
+export const livepeerClient = createReactClient({
+  provider: studioProvider({
+    apiKey: process.env.NEXT_PUBLIC_LIVEPEER_API_KEY as string,
```

Comment:
we're using [t3-env](https://env.t3.gg/) for env variable validation, so you should instantiate it rather than the regular `process` object. Also, avoid type casting env variables and prefer to handle their edge cases correctly.
```suggestion
    apiKey: env.NEXT_PUBLIC_LIVEPEER_API_KEY,
```